src/@next/Tag/Tag.tsx

```diff
@@ -12,7 +12,7 @@ import {
 export interface TagProps extends React.HTMLAttributes<HTMLDivElement> {
   children?: React.ReactNode;
   value?: string;
-  onRemove?: (() => void) | null;
+  onRemove?: (() => void) | null | void;
```

Comment:
shouldn't this be `onRemove?: () => void | null` ? or is it accepting function as an argument?

Reply:
yes, it is accepting function